app/pages/lab/workflow.cjsx

```diff
@@ -105,6 +106,14 @@ EditWorkflowPage = React.createClass
                       <small><strong>Drawing</strong></small>
                     </button>
                   </AutoSave>{' '}
+                  {if @canUseTask(@props.project, "text")
+                    <AutoSave resource={@props.workflow}>
+                      <button type="submit" className="minor-button" onClick={@addNewTask.bind this, 'text'} title="Text tasks: the volunteer writes free-form text into a dialog box.">
+                        <i className="fa fa-pencil fa-2x"></i>
+                        <br />
+                        <small><strong>Text</strong></small>
+                      </button>
+                    </AutoSave>}
```

Comment:
Can we keep the spaces characters (`{' '}`) between these buttons? Otherwise they're inconsistently jammed together.